tests/flytekit/unit/taskplugins/sagemaker/test_training.py

```diff
@@ -0,0 +1,85 @@
+import os
+import tempfile
+from typing import TypeVar
+
+import pytest
+
+from flytekit import task, metadata, kwtypes
+from flytekit.annotated.context_manager import RegistrationSettings, Image, ImageConfig
+from flytekit.taskplugins.sagemaker import SagemakerTrainingJobConfig, SagemakerBuiltinAlgorithmsTask, \
+    TrainingJobResourceConfig, AlgorithmSpecification, AlgorithmName
+from flytekit.types import FlyteFile
+
+
+def _get_reg_settings():
```

Comment:
maybe we should have a testutils and move this there, considering the amount of times we create a RegistrationSettings obj for tests :)

Reply:
ya